### DISCUSSION TITLE
Forgot email for my shortened link

### DISCUSSION THREAD

**Discussion by ACXCloud** [dubinc/dub#254] [General]
https://dub.sh/gclone

this is my shortened link I forgot what email I used to signup can you help me out ?

**Comment by ACXCloud** [comment]
Please any update ?

**Reply by gabefletch** [reply]
I've opened a new issue (#395) about this. I realize that this is a pretty old discussion but it seems important as other users could very well have the same difficulty with account recovery in the future.

**Comment by steven-tey** [comment]
Sorry for the late response here – @ACXCloud if you can send us an email at support@dub.co, I can get you the email you used for that link!
